app/controllers/application_controller.rb

```diff
@@ -45,8 +45,12 @@ def catch_flexirest_exceptions
          Flexirest::HTTPServerException,
          Flexirest::HTTPNotFoundClientException => e
 
-    errors_hash = e.result.Errors.to_a.map do |err|
-      { "title": err.to_hash['Message'] }
+    if e.result.respond_to? :Errors
+      errors_hash = e.result.Errors.to_a.map do |err|
+        { "title": err.to_hash['Message'] }
+      end
+    else
+      errors_hash = { "title": e.result.to_hash['errors']&.items&.first&.message }
```

Comment:
Should we just repeat what we have above, but with the different casing?

i.e.

```ruby
errors_hash = e.result.errors.to_a.map do |err|
  { "title": err.to_hash['message'] }
end
```